### PR TITLE
Solution 8 docker build and push

### PR DESCRIPTION
**Pull request by mrblonde91**

### CODE REVIEW

File: .github/workflows/solution8-api-push-image.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    # Publish `main` as Docker `latest` image.
+    branches:
+      - main
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+env:
+  # TODO: Change variable to your image's name.
+  IMAGE_NAME: silo-host-test
```
Comment (PiotrJustyna) [inline .github/workflows/solution8-api-push-image.yml:14]: This should probably reflect the API name somehow.

File: .github/workflows/solution8-host-push-image.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    # Publish `main` as Docker `latest` image.
+    branches:
+      - main
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+env:
+  # TODO: Change variable to your image's name.
+  IMAGE_NAME: silo-api-test
```
Comment (PiotrJustyna) [inline .github/workflows/solution8-host-push-image.yml:14]: Yep, I think the names got swapped :)

**Review by PiotrJustyna — APPROVED** [review]
Just minor comments.

File: .github/workflows/solution8-host-push-image.yml (hunk 2 above)
Comment (mrblonde91) [reply .github/workflows/solution8-host-push-image.yml:14]: oops

**Review by PiotrJustyna — APPROVED** [review]

**Pull request merged**